specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-08-01/workbooks_API.json

```diff
@@ -0,0 +1,741 @@
+          "additionalProperties": {
+            "type": "string"
+          },
+          "type": "string",
+          "description": "Resource etag"
+        }
+      }
+    },
+    "WorkbooksListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Workbook"
+          },
+          "description": "An array of workbooks."
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      },
+      "description": "Workbook list result."
+    },
+    "Workbook": {
+      "description": "An Application Insights workbook definition.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/WorkbookResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Metadata describing a workbook for an Azure resource.",
+          "$ref": "#/definitions/WorkbookProperties"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData",
+          "readOnly": true
+        }
+      }
+    },
+    "WorkbookProperties": {
+      "description": "Properties that contain a workbook.",
+      "type": "object",
+      "required": [
+        "displayName",
+        "category",
+        "serializedData"
+      ],
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The user-defined name (display name) of the workbook."
+        },
+        "serializedData": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "Configuration of this particular workbook. Configuration data is a string containing valid JSON"
+        },
+        "version": {
+          "type": "string",
+          "description": "Workbook schema version format, like 'Notebook/1.0', which should match the workbook in serializedData"
+        },
+        "timeModified": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time in UTC of the last modification that was made to this workbook definition.",
+          "readOnly": true
+        },
+        "category": {
+          "type": "string",
+          "description": "Workbook category, as defined by the user at creation time."
+        },
+        "tags": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Being deprecated, please use the other tags field"
+        },
+        "userId": {
+          "type": "string",
+          "description": "Unique user id of the specific user that owns this workbook.",
+          "readOnly": true
+        },
+        "sourceId": {
+          "type": "string",
+          "description": "ResourceId for a source resource."
+        },
+        "storageUri": {
```

Comment:
>storageUri [](http://example.com/codeflow?start=9&length=10)

Please could you rename to 'storageAccountId' ?

Reply:
although i would agree with this, this was created for it was first a uri. this would require backend change and backward compat with existing version that it's not worth changing at this time